depmapomics/loading.py

```diff
@@ -814,14 +815,10 @@ def load(
     maxage,
     baits,
     stype,
+    trackerobj=None,
```

Comment:
Seems like there's a fair number of places where trackerobj is defined as optional.